### DISCUSSION TITLE
Improve modulate documentation

### DISCUSSION THREAD

**Discussion by hyperknot** [ImageMagick/ImageMagick#3305] [Development]
I've run into a few issues with modulate's documentation: https://imagemagick.org/script/command-line-options.php#modulate

1. `-set` vs `-define` as written here: https://github.com/ImageMagick/ImageMagick/discussions/3303
2. The list of colorspaces is missing `LCHab` (and possibly others?)
3. Explain how to make hue-shift, `100,100,x` needs to be specified.
4. Explain that when doing hue-shift, the differing colorspace options are: `HCL`, `HCLp`, `HSL`, `LCHuv`, `LCHab`.

**Comment by snibgo** [comment]
Although you can use any colorspace, the operation is only really for HCL, HCLp, HSB, HSI, HSL, HSV, HWB, LCH, LCHab and LCHuv. If you use any other colorspace, the effect is the same as HSL. They can all be used for shifting hue.

**Reply by hyperknot** [reply]
> They can all be used for shifting hue.

They can, but I just tested with md5sum on a tga output and it turned out that:
1. HSB=HSI=HSL=HSV=HWB and 
2. LCH=LCHuv

1. is probably only true for hue-shift, while 2. is I guess an alias.
